src/plone/restapi/serializer/nextprev.py

```diff
@@ -23,8 +24,9 @@ def __init__(self, context):
 class NextPrevious:
     """Facade with more pythonic interface"""
 
-    def __init__(self, context):
+    def __init__(self, context, request):
```

Comment:
This is not a backwards-compatible change. You can use getRequest from zope.globalrequest to get the request without changing the parameters here.

Reply:
Done in https://github.com/plone/plone.restapi/pull/1355/commits/ff979592b7c204a54c419ef6879e2011b3b74ab4